### PR TITLE
[Readme] Clarify that csslint is a npm package

### PR DESCRIPTION
**Pull request by surfer190**
It isn't that clear that `csslint` is a npm package, so I added instructions for getting that installed and formatted the readme a bit

### CODE REVIEW

**Comment by kmmbvnr** [comment]
Please also add this for other npm packages

**Comment by surfer190** [comment]
Are there any other npm packages? Wasn't `jshint` removed?

**Comment by kmmbvnr** [comment]
ups, seems it was accidentally removed.

-- ``django_jenkins.tasks.run_jshint``
 -
 -  Runs jshint tools over ``<app>/static/*/*.js`` files.
 -  Creates Pylint compatible report for Jenkins
 -
 -  You should have the jshint installed

**Comment by surfer190** [comment]
Looks like you removed it: 1c4ec1ceaeeaa8823f0c10a5e74a85199eb94c40

I have made the changes now

File: README.rst [hunk 1]
```diff
@@ -138,7 +154,7 @@ GIT Version
 * Flake8 >= 2.5.0 support
 
 
-0.18.0 1985-10-26
+0.18.0 2015-10-26
```
Comment (kmmbvnr) [inline README.rst]: no

Comment (surfer190) [inline README.rst]: why?

Comment (kmmbvnr) [inline README.rst]: first of all, if you don't understand, you should not try to fix.

**Pull request closed without merging**